spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/cassandra/CassandraProperties.java

```diff
@@ -26,12 +26,14 @@
 import com.datastax.driver.core.policies.RetryPolicy;
 
 import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.data.cassandra.config.SchemaAction;
```

Comment:
I am afraid we can't do that. Plain cassandra support may work without Spring Data Cassandra. I just had a chat with @philwebb about this and it can be tricky.

I think our best option is to create a meta-data entry manually (that's what we usually do when we need to retrieve a value type that may not be on the classpath). I also noticed that we don't really have a service that would wrap the converters that are being used by the  `ConfigurationPropertiesBindingPostProcessor`. I've created #4890 to track that.